bin/test.sh

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env bash
+set -e +o pipefail
```

Comment:
@DmitryBespalov could it be a reason why unit tests can not run on Jenkins?

Reply:
I think, this is the reason:
![Screenshot 2021-07-01 at 11 45 26](https://user-images.githubusercontent.com/284056/124103729-dda20d80-da61-11eb-9266-9935ed886265.png)